### DISCUSSION TITLE
[Task]: Deploy Demo

### DISCUSSION THREAD

**Discussion by ggoodd-any** [open-edge-platform/anomalib#1141] [Feature Requests]
### What is the motivation for this task?

These defect detection models usually need to be deployed to some Edge device. I hope to have a simple demo to guide beginners in deploying models.

### Describe the solution you'd like

Provide a Python code that uses the onnx model for inference.

Provide a C++code that uses the onnx model for inference.

**Comment by samet-akcay** [comment]
Hi @qqqhhh-any, this is something we have been recently trying to address. You will hopefully hear from us soon.

**Reply by Kyle-fang** [reply]
Hello! I used onnxruntime to reason about onnx model and got the output as an anomaly map of [1, 1, 256, 256], but I want to know if the current image is an anomaly or a person normal sample what should I do? Is there code to accomplish this?

**Reply by blaz-r** [reply]
Answered in #1031.
